### DISCUSSION TITLE
Need help with Collision Filter Extension and Text Layer

### DISCUSSION THREAD

**Discussion by ssriv48** [visgl/deck.gl#8748] [Q&A]
I am rendering Text Layer (along with other layers) and need to use CollisionFilterExtension in my app. When I provide a function for getPixelOffset property, the texts all disappear. If I disable the CollisionFilterExtension, the texts are rendered fine with proper offset, in a nutshell, I can't get the CollisionFilterExtension and the getPixelOffset to work together, individually they both work fine. 

Please help, even if someone could suggest changes in the TextLayer code or CollisionFilterExtension code, it's fine with me, I can copy and customize the extension / layer.

Thanks

**Comment by amuedespacher** [comment]
Hi @ssriv48, could you please post a reproducible code (e.g. a CodePen), so that one can look into it?
